CDP4Dal/DAL/Dal.cs

```diff
@@ -62,11 +62,11 @@ protected Dal()
             this.MetaDataProvider = StaticMetadataProvider.GetMetaDataProvider;
             this.SetCdpVersion();
         }
-        
+
         /// <summary>
         /// Gets the supported version of the data-model
         /// </summary>
-        public Version DalVersion { get; private set; }
+        public Version DalVersion { get; set; }

```

Comment:
is changing the accesor to public necessary? if you change it here, that does not mean the serializer version gets updated i think. I propose to revert, or to make sure that the serializer version is updated as well.. We might be introducing complexity and unexpected behavior?

Reply:
Changed to protected.